server/app/views/applicant/ProgramIndexView.java

```diff
@@ -350,6 +367,21 @@ private PTag programCardApplicationStatus(
                 .withClasses("text-xs", "font-medium"));
   }
 
+  private PTag eligibilityTag(Messages messages, boolean isEligible) {
+    Icons icon = isEligible ? Icons.CHECK_CIRCLE : Icons.INFO;
+    String color = isEligible ? "bg-blue-100" : "bg-gray-200";
+    String tagClass =
+        isEligible ? ReferenceClasses.ELIGIBLE_TAG : ReferenceClasses.NOT_ELIGIBLE_TAG;
+    String tagText =
+        isEligible
+            ? MessageKey.TAG_MAY_QUALIFY.getKeyName()
+            : MessageKey.TAG_MAY_NOT_QUALIFY.getKeyName();
+    return p().withClasses(tagClass, "border", "rounded-lg", "px-2", "py-1", "mb-4", color)
+        .with(
+            Icons.svg(icon).withClasses("inline-block").withStyle("width: 18px; height: 18px;"),
```

Comment:
**question (blocking):** were the tailwind built-in classes not able to implement the design?

Reply:
Could you give more info? I implemented this similar to some Icons in ViewUtils (e.g. https://github.com/civiform/civiform/blob/main/server/app/views/ViewUtils.java#L107)

Reply:
Ah that was in reference to using the inline styles `.withStyle("width: 18px; height: 18px;")` vs uses the tailwind width and height classes

Reply:
Ah makes sense! That height / width isn't supported - added a comment to make that clear.